cmake/Kokkos_Version_Info.hpp

```diff
@@ -14,8 +14,8 @@
 //
 //@HEADER
 
-#ifndef GIT_VERSION_H
-#define GIT_VERSION_H
+#ifndef KOKKOS_GIT_VERSION_INFO_H
```

Comment:
Unrelated fix - carry over from previous PR.

Reply:
I will close my eyes this time but in the future do this in a separate PR